tests/hpc/before_test.pm

```diff
@@ -23,18 +23,19 @@ sub run {
     # disable packagekitd
     pkcon_quit();
 
-    my $repo     = get_required_var('HPC_REPO');
-    my $reponame = get_required_var('HPC_REPONAME');
-    zypper_call("ar -f $repo $reponame");
-    assert_script_run "zypper lr | grep $reponame";
+    if (testapi::get_var('HPC_REPO')) {
```

Comment:
Why is the testapi prefix necessary?

Reply:
I think this way is more easy to refactor code later , if all functions would use this you will not get into trouble which I get when I was creating version_utils

Reply:
Does not make sense to me. os-autoinst-distri-opensuse clearly relies on the functions within the test API and should not need to prefix any method from there

Reply:
and for me it make sense , even you don't know all functions and from where they come from , not speaking about newcomers .  for me it is more effective way then having 

`use version_utils qw(is_sle is_leap);`

Reply:
prefixes removed